### PR TITLE
Makes Nitrous Oxide Visible on Air Alarm, PDA, and Analyzer

### PR DESCRIPTION
**Pull request by silverplatedelta**
## What Does This PR Do
Tweaks N2O to no longer be a trace gas. It now appears as nitrous oxide on scanners, PDAs, and air alarms. it no longer appears as other or unknown

## Why It's Good For The Game
Nitrous oxide is a known gas, it should be able to be scanned by standard NT systems. This should also help reduce player confusion.

## Images of changes
Image of analyzer output:
Step1
Image of air alarm output:
Step2

## Testing
Spawn in, open a can of nitrous in atmos. Scan atmosphere with various devices. Confirm no runtimes where thrown.

## Changelog
:cl:
tweak: Nitrous oxide now appears like other gasses on air alarms and scanners.
/:cl:

### CODE REVIEW

**Review by Fox-McCloud — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Comment by farie82** [comment]
TGUI conflict. If ya rebuild the TGUI bundle then this PR can be merged

File: code/modules/atmospherics/machinery/airalarm.dm [hunk 1]
```diff
@@ -1111,13 +1127,6 @@
 		new /obj/item/stack/cable_coil(loc, 3)
 	qdel(src)
 
-/obj/machinery/alarm/AltClick(mob/user)
-	if(Adjacent(user) && allowed(user) && !wires.is_cut(WIRE_IDSCAN))
-		locked = !locked
-		to_chat(user, "<span class='notice'>You [locked ? "lock" : "unlock"] the Air Alarm interface.</span>")
-	else
-		to_chat(user, "<span class='warning'>Access denied.</span>")
-
```
Comment (farie82) [inline code/modules/atmospherics/machinery/airalarm.dm:1120]: This shouldn't be removed as it got added by a recent PR

File: code/game/objects/items/devices/scanners.dm [hunk 2]
```diff
@@ -397,7 +397,7 @@ REAGENT SCANNER
 	if(!user.incapacitated() && Adjacent(user))
 
 		if(cooldown)
-			to_chat(user, "<span class='warning'>[src]'s barometer function is preparing itself.</span>")
+			to_chat(user, "<span class='warning'>[src]'s barometer function is prepraring itself.</span>")
```
Comment (farie82) [inline code/game/objects/items/devices/scanners.dm:400]: Same here

File: code/game/objects/items/devices/scanners.dm [hunk 3]
```diff
@@ -57,9 +57,6 @@ REAGENT SCANNER
 			var/mutable_appearance/MA = new(O)
 			MA.alpha = 128
 			MA.dir = O.dir
-			if(MA.layer < TURF_LAYER)
```
Comment (farie82) [inline code/game/objects/items/devices/scanners.dm:60]: Stuff in this file as well seems to be incorrectly merged

**Review by farie82 — CHANGES_REQUESTED** [review]
You can reset to commit [6ff1999](https://github.com/ParadiseSS13/Paradise/pull/20891/commits/6ff19991fa30d33df4dab8539eea677278d794b4) after which you can rebase/merge master into your branch. Feel free to ask for help on discord if ya need it

File: code/modules/atmospherics/machinery/airalarm.dm (hunk 1 above)
Comment (silverplatedelta) [reply code/modules/atmospherics/machinery/airalarm.dm]: Rebased branch based on an updated one with the same changes (I think)

File: code/game/objects/items/devices/scanners.dm (hunk 2 above)
Comment (silverplatedelta) [reply code/game/objects/items/devices/scanners.dm]: Rebased branch based on an updated one with the same changes (I think)

File: code/game/objects/items/devices/scanners.dm (hunk 3 above)
Comment (silverplatedelta) [reply code/game/objects/items/devices/scanners.dm]: Rebased branch based on an updated one with the same changes (I think)

**Review by farie82 — APPROVED** [review]

**Pull request merged**